README.md

```diff
@@ -148,6 +148,14 @@ If you have your own application to run, you can follow the instructions startin
 [Configure a Fluo application][3] section of the Fluo production setup instructions to configure, initialize, 
 and start your application.  To automate these steps, you can mimic the scripts of example Fluo applicaitons above.
 
+Customize your cluster
+----------------------
+
+After `zetten setup` is run, users can install additional software on the cluster using their own Ansible playbooks. In their
+Ansible playbooks, users can use common cluster variables that are configured by Zetten in the `[all:vars]` section of the 
```

Comment:
No, the hosts/inventory file also specifies the hosts for workers, zookeeper, namenode, etc.  The `[all:vars]` section is the only section that has variables.  See http://docs.ansible.com/ansible/intro_inventory.html for more info on the hosts file.